index.html

```diff
@@ -4,7 +4,7 @@
 
 	<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js">
 	</script>
-	<title></title>
+	<title>My Form</title>
```

Comment:
This should actually be a different title.
```suggestion
	<title>Survey Form Page</title>
```